### PR TITLE
[LIMINAL-85]: Liminal delete

### PR DESCRIPTION
**Pull request by naturalett**

### CODE REVIEW

**Review by aviemzur — CHANGES_REQUESTED** [review]
Looks good. Could use unit tests for creation and deletion of the volumes to see if they were appropriately created or deleted from k8s.

**Comment by naturalett** [comment]
unittests were added

File: tests/liminal/kubernetes/test_volume_util.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        }
+
+    def test_volume_config(self):
+        volumes_config = volume_util.get_volume_configs(self.config, ".")
+        self.assertEqual(
+            str(self.config['volumes'][0]),
+            str(volumes_config[0]))
+
+    def test_create_volume(self):
+        self._create_volumes()
```
Comment (aviemzur) [inline tests/liminal/kubernetes/test_volume_util.py:52]: Please add a call to delete at the beginning of this test

File: tests/liminal/kubernetes/test_volume_util.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        matching_volumes = volume_util._list_persistent_volumes(self.config['volumes'][0]['volume'])
+        self.assertTrue(self.config['volumes'][0]['volume'] in matching_volumes[0]['metadata']['name'],
+                        self.config['volumes'][0]['claim_name'] in matching_volumes[0]['spec']['claim_ref']['name'])
+                        
+    def test_delete_volume(self):
+        self._delete_volumes()
```
Comment (aviemzur) [inline tests/liminal/kubernetes/test_volume_util.py:58]: Please add a call to create at the beginning of this test

**Review by aviemzur — CHANGES_REQUESTED** [review]

**Review by aviemzur — APPROVED** [review]

**Pull request merged**